thunderbird/processes/wps_say_hello.py

```diff
@@ -0,0 +1,59 @@
+from pywps import Process, LiteralInput, LiteralOutput, UOM
```

Comment:
This file has also been removed from `master`. Your git history seems to be either out of date or jumbled a bit in here.